packages/cozy-harvest-lib/src/services/softDeleteOrRestoreAccounts.js

```diff
@@ -0,0 +1,85 @@
+import logger from './logger'
+import {
+  decryptString,
+  getOrganizationKey,
+  fetchAccountsForCipherId,
+  fetchKonnectorFromAccount,
+  fetchTriggersFromAccount,
+  updateAccountsAuth
+} from './utils'
+import get from 'lodash/get'
+import { ensureTrigger } from '../connections/triggers'
+import { translate as t } from 'cozy-ui/transpiled/react/I18n'
```

Comment:
🤔 translate is a higher order function that decorates a React component so that it receives a `t` function in its props. It does not directly "translate" a locale string. It's the t function that should be passed to ensureTrigger, not translate.

To have a t function you can do

```
const Polyglot = require('node-polyglot')
const supportedLocales = ['en', 'fr', 'es']
const getT = () => {
  const locale = supportedLocales.find(l => l === process.env.COZY_LOCALE) || 'en'
  const locales = require(`../src/locales/${locale}.json`)
  const polygot = new Polyglot()
  polyglot.extend(locales)
  return polyglot.t.bind(polyglot)
}
```

Reply:
Ok, thanks